generators/client/index.js

```diff
@@ -197,6 +197,12 @@ module.exports = class JHipsterClientGenerator extends BaseBlueprintGenerator {
         this.loadTranslationConfig();
       },
 
+      checkMicrofrontend() {
+        if (this.microfrontend && !this.clientFrameworkAngular) {
```

Comment:
if this is a new option, then this should be added to the JDL app validation

Reply:
In my opinion Microfrontend is too experimental to be an official jdl option.

Reply:
alright then